components/google_cloud/actions/switch-instance-boot-status/switch-instance-boot-status.mjs

```diff
@@ -0,0 +1,63 @@
+import googleCloud from "../../google_cloud.app.mjs";
+
+export default {
+  name: "Switch Instance Boot Status",
+  version: "0.0.1",
+  key: "google_cloud-switch-instance-boot-status",
+  type: "action",
+  description: "Switch a virtual machine instance boot status to start or stop it. [See the documentation](https://cloud.google.com/compute/docs/instances/stop-start-instance)",
+  props: {
+    googleCloud,
+    zone: {
+      propDefinition: [
+        googleCloud,
+        "zoneName",
+      ],
+    },
+    instanceName: {
+      propDefinition: [
+        googleCloud,
+        "instanceName",
+        ({ zone }) => ({
+          zone,
+        }),
+      ],
+    },
+    newInstanceStatus: {
+      label: "New Instance Status",
+      description: "Select if the instance should be either switched to started or stopped status",
+      type: "string",
+      options: [
+        "start",
+        "stop",
+      ],
+    },
+    waitCompletion: {
+      label: "Wait Completion",
+      description: "Wait until the end of the operation",
```

Comment:
```suggestion
      description: "Wait until the end of the operation"
```

Reply:
This one is definitely a false positive @dylburger , as removing the comma here would break the code syntax. There are several other cases in this file as well